smooth/components/component_battery.py

```diff
@@ -266,5 +267,7 @@ def update_states(self, results, sim_params):
                     # Initialize a.n array that tracks the state SoC
                     self.states["soc"] = [None] * sim_params.n_intervals
                 # Check if this result is the state of charge.
-                self.soc = df_storage[i_result][0] / self.battery_capacity
+                # todo: Remove hack + 0.005 and find better solution
+                # self.soc = df_storage[i_result][0] / self.battery_capacity
+                self.soc = (df_storage[i_result][0]+0.005) / self.battery_capacity
```

Comment:
This fix is meant for the MPC use, which is not yet merged. Furthermore, it might be necessary to discuss this change with everybody.